packages/ember-testing/lib/test/waiters.js

```diff
@@ -7,7 +7,8 @@ const callbacks = [];
 /**
    This allows ember-testing to play nicely with other asynchronous
    events, such as an application that is waiting for a CSS3
-   transition or an IndexDB transaction.
+   transition or an IndexDB transaction. The waiter runs periodically
```

Comment:
The work is done by way of `wait()`, but all async helpers run `app.testHelpers.wait()` at the end by default (you shouldn't have to `return app.testHelpers.wait()` manually in an async test helper).